libs/deal-ticket/src/components/deal-ticket/deal-ticket.tsx

```diff
@@ -54,13 +53,11 @@ export const DealTicket = ({
     mode: 'onChange',
     defaultValues: getDefaultOrder(market),
   });
-
-  const orderType = watch('type');
-  const orderTimeInForce = watch('timeInForce');
+  const order = watch();
```

Comment:
question, non-blocking: watching whole object instead certain properties wouldn't provide more re-renders?

Reply:
it is intentional, I do need all fields inside the order.